### PR TITLE
chore(aio): blank line before each list-item; - becomes *

### PR DESCRIPTION
**Pull request by wardbell**
For conversion to new `aio`, to satisfy the peculiarities of the Rho markdown processor, we must

* put blank line before the beginning of a list of any kind (ul, ol, dl)

* replace dash (-) with asterisk (*) because in Rho the dash means `<dl><dd>` and we always meant it to be a `<ul><li>`.

This change may trigger some unwelcome presentation in `angular.io` (and in `aio` too) especially w/r/t to nested lists (typically Table of Contents). We'll have to deal with those case-by-case.

### CODE REVIEW

File: public/docs/ts/latest/guide/forms.jade [hunk 1]
```diff
@@ -128,16 +128,17 @@ figure.image-display
...
 
-  - The code imports the Angular core library and the `Hero` model you just created.
-  - The `@Component` selector value of "hero-form" means you can drop this form in a parent template with a `<hero-form>` tag.
-  - The `templateUrl` property points to a separate file for the template HTML.
-  - You defined dummy data for `model` and `powers`, as befits a demo.
+  * The code imports the Angular core library and the `Hero` model you just created.
+  * The `@Component` selector value of "hero-form" means you can drop this form in a parent template with a `<hero-form>` tag.
+  * The `templateUrl` property points to a separate file for the template HTML.
+  * You defined dummy data for `model` and `powers`, as befits a demo.
   Down the road, you can inject a data service to get and save real data
```
Comment (Foxandxss) [inline public/docs/ts/latest/guide/forms.jade]: blank line in here

File: public/docs/ts/latest/guide/lifecycle-hooks.jade [hunk 2]
```diff
@@ -522,8 +529,9 @@ figure.image-display
 .l-sub-section
   :marked
     The telltale signs of *content projection* are twofold:
-      - HTML between component element tags.
-      - The presence of `<ng-content>` tags in the component's template.
+
+      * HTML between component element tags.
+      * The presence of `<ng-content>` tags in the component's template.
 a#aftercontent-hooks
```
Comment (Foxandxss) [inline public/docs/ts/latest/guide/lifecycle-hooks.jade]: blank line

File: public/docs/ts/latest/guide/ngmodule.jade [hunk 3]
```diff
@@ -25,9 +25,10 @@ block includes
...
 
   <!-- CF: The titling for tables of contents in the advanced chapters is inconsistent:
-  - some are titled "Contents" while others are titled "Table of Contents" (should probably be sentence case as it's an H2
-  - some headings are H2, some are H3
-  - some pages don't have tables of contents
+
+  * some are titled "Contents" while others are titled "Table of Contents" (should probably be sentence case as it's an H2
+  * some headings are H2, some are H3
+  * some pages don't have tables of contents
   I didn't make changes here as I'm not sure what the correct style is.
```
Comment (Foxandxss) [inline public/docs/ts/latest/guide/ngmodule.jade]: blank line

File: public/docs/ts/latest/guide/server-communication.jade [hunk 4]
```diff
@@ -19,37 +19,46 @@ block includes
...
     <li> [The WikiComponent](#wikicomponent)</li>
   </ul>
+
   * [A wasteful app](#wasteful-app)
   <li> [More fun with Observables](#more-observables)
       <ul>
         <li> [Create a stream of search terms](#create-stream)</li>
         <li> [Listen for search terms](#listen-for-search-terms)</li>
       </ul>
   </li>
```
Comment (Foxandxss) [inline public/docs/ts/latest/guide/server-communication.jade]: any reason all of this is html? But they seem to render fine. Moving on...

Comment (Foxandxss) [inline public/docs/ts/latest/guide/server-communication.jade]: A block in here? I thought I removed them all.

Comment (wardbell) [inline public/docs/ts/latest/guide/server-communication.jade]: Markdown sucks with lists. Well known. I guess I gave up trying to make it work and switched.
We'll have to do that in `aio` quite a bit more I fear.

File: public/docs/ts/latest/guide/lifecycle-hooks.jade (hunk 2 above)
Comment (wardbell) [inline public/docs/ts/latest/guide/lifecycle-hooks.jade]: Don't think it matters but ok

**Pull request merged**